src/unix/linux/disk.rs

```diff
@@ -423,7 +429,39 @@ fn get_all_list(container: &mut Vec<Disk>, content: &str, refresh_kind: DiskRefr
             &procfs_disk_stats,
             refresh_kind,
         ));
+
+        // If the disk is mounted, remove it from the list
+        unmounted.retain(|disk| disk != fs_spec);
     }
+
+    // Process the unmounted disks
+    for disk in unmounted {
+        // The mount point should be left empty unless it is ZRAM
+        let mount_point = match disk.contains("zram") {
+            true => "[SWAP]",
+            _ => "",
+        };
+        let mount_point = Path::new(mount_point);
+
+        // File system is Unknown or swap
+        // TODO: Find a way to figure out the file system (like lsblk)
+        let file_system = match disk.contains("zram") {
+            true => "swap",
+            _ => "Unknown",
+        };
+
+        container.push(new_disk(
+            disk.as_ref(),
+            mount_point,
```

Comment:
With this new feature, I think `mount_point` should become an enum looking like this:

```rust
enum MountPoint {
    Path(Path)
    Swap,
    Unmounted,
}
```